src/prometheus-net.Contrib/Diagnostics/AspNetCoreListenerHandler.cs

```diff
@@ -22,12 +22,37 @@ private static class PrometheusCounters
             public static readonly Counter AspNetCoreRequestsErrors = Metrics.CreateCounter(
                 "aspnetcore_requests_errors_total",
                 "Total HTTP requests received errors.");
+
+            public static readonly Counter AspNetCoreRequestsTotal = Metrics.CreateCounter(
+                "aspnetcore_requests_total",
+                "Total Requests.",
+                new CounterConfiguration
+                {
+                    LabelNames = new[] { "code", "method", "route" }
+                });
+
+            public static readonly Gauge AspNetCoreRequestsCurrentTotal = Metrics.CreateGauge(
+                "aspnetcore_requests_current_total",
```

Comment:
Because this is a Gauge, I think we can call the metric `aspnetcore_requests_in_progress`

"_accumulating count has total as a suffix, in addition to the unit if applicable._"
https://prometheus.io/docs/practices/naming/

Reply:
I think you also need to decrease the gauge if an exception occurs.

Reply:
Done

Reply:
Should this be updated, then?

https://github.com/alexvaluyskiy/prometheus-net-contrib/blob/bd055c5114538a3cde76465640178c6d0b11751b/src/prometheus-net.Contrib/EventListeners/Adapters/PrometheusAspNetCoreCounterAdapter.cs#L11